packages/block-library/src/block/test/edit.native.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { cleanup, fireEvent, waitFor } from '@testing-library/react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import { fetchRequest } from '@wordpress/react-native-bridge';
+
+/**
+ * Internal dependencies
+ */
+import { initializeEditor } from 'test/helpers';
+import { getReusableBlock } from './fixture';
+
+beforeAll( () => {
+	// Setup API fetch as it's done when initializing the editor
+	require( '@wordpress/react-native-editor/src/api-fetch-setup' ).default();
+	// Register all core blocks
+	require( '@wordpress/block-library' ).registerCoreBlocks();
```

Comment:
Following up here with more thoughts. Here is an example of [Cover edit registering only the required blocks](https://github.com/WordPress/gutenberg/blob/65a9a15079592cf07c629b14ec9201072df332df/packages/block-library/src/cover/test/edit.native.js#L65-L74). 

In regards to which approach is the best path forward, my concerns come down to: 

1. Does registering _all_ blocks add complexity to the test that would make debugging breakages difficult?
2. Will the tests be meaningfully slower by registering _all_ blocks?

The first question is likely more a gut feeling. From my testing there is not a noticeable speed difference to either approach, so I don't believe it is relevant. 

So, registering all block may be just fine.

Reply:
>Would it make more sense to register only the blocks required for this testing environment or is that too cumbersome?

Good question! To be honest, I don't think we need to register all the blocks, it would be enough to register the required ones for the test. In reusable block's case, since they can contain any block I decided to import all of them but I could reduce it to the ones referenced in the fixture.

>1. Does registering all blocks add complexity to the test that would make debugging breakages difficult?

That's a good point, I'm not sure if having all blocks would add too much complexity because it's only registration. I think it could affect debugging if the breakage is caused in the initialization code of the block but probably that's something not likely to happen.

>2. Will the tests be meaningfully slower by registering all blocks?

I haven't benchmarked this but my impression is that block registration time is quite low but I can't really assure it.

>The first question is likely more a gut feeling. From my testing there is not a noticeable speed difference to either approach, so I don't believe it is relevant.
>
>So, registering all block may be just fine.

In my opinion, I think it depends on the test, probably for testing a single block won't be necessary but for tests like adding blocks or HTML output check, we would need all of them.

Reply:
> In my opinion, I think it depends on the test, probably for testing a single block won't be necessary but for tests like adding blocks or HTML output check, we would need all of them.

Make sense. I think your current approach works well for this test.